### PR TITLE
Remove errant handout kits

### PR DESCRIPTION
**Pull request by squirrelo**
This adds a listing of all handout kits in the system, using DataTables, that can be exported to excel format or used to delete errant kits. 

Requires #55

### CODE REVIEW

File: knimin/lib/data_access.py [hunk 1]
```diff
@@ -2032,6 +2053,32 @@ def search_handout_kits(self, term):
...
+            Each kit as a dict with all available info. Barcodes are a list
+            under the barcode key.
+        """
+        sql = """SELECT kit_id, verification_code, swabs_per_kit, created_on,
+                 array_agg(barcode) as barcodes
+                 FROM ag.ag_handout_kits
+                 JOIN ag.ag_handout_barcodes USING (kit_id)"""
+        sql_args = None
+        if kit_ids is not None:
+            sql += 'WHERE kit_ids IN %s'
```
Comment (josenavas) [inline knimin/lib/data_access.py]: Missing a space before `WHERE`. Probably better to create `sql` as a list and then do a `' '.join(sql)` at the end. I think list appending is cheaper than string concatenation.

**Comment by josenavas** [comment]
I'm gonna be super-paranoic here. Will it make sense to write the deleted information into a log file or something so we can recover in case of undesired deletion? I may be missing something, but the delete shouldn't be checking that the kits have been already verified?

**Comment by squirrelo** [comment]
This only deletes handout kits, so there's no verification needed that is programatic, only human to make sure they haven't been shipped yet. I agree on a log of some sort, but unsure how. Maybe just a TSV we keep appending to similar to other logging software, with the info for the given kit(s)?

**Comment by josenavas** [comment]
Ok - yes the TSV that we append should be good enough - just add also the timestamp so it is easily searchable. I would also suggest to put some kind of size limitation (i.e. if the file is X bytes, close, add an incremental suffix, and open a new one).

**Pull request closed without merging**